UI improvements

adalon opened the pull request.

adalon committed:
Renamed "Build NuGet Package" command to "Create NuGet Package"

And disabling the DefaultInvisible for now until figure out why it's not working

adalon committed:
Added EditNuGetPackageMetadata view

And also added VS theme to be used by the WPF dialogs

adalon committed:
Reading and writing the values of the PackageMetadata editor

adalon committed:
Showing the package editor if the nuget is not installed

When the "Create NuGet Package" command is executed

adalon committed:
Improved AddPlatformImplementation and Multiplatform views

By inheriting form VS.DialogWindow, using theme and fixed layout

adalon committed:
Fixed constants

kzu commented on src/VisualStudio/NuGet.Packaging.VisualStudio/Views/PackageMetadataViewModel.cs:
```diff
@@ -0,0 +1,119 @@
+
+namespace NuGet.Packaging.VisualStudio
+{
+	class PackageMetadataViewModel
```
Some properties are incorrect. Please ensure they match the ones used in the [build targets](https://github.com/NuGet/NuGet.Build.Packaging/blob/dev/src/Build/NuGet.Build.Packaging.Tasks/NuGet.Build.Packaging.targets#L92-L113)

adalon replied:
Fixed

kzu reviewed (changes requested):
Other than fixing the property names so they match the targets, it looks great!

adalon committed:
Fixed package property names to match the ones used by the build targets

<PackageTargetPath Include="$([System.IO.Path]::GetFullPath('$(PackageTargetPath)'))">
				<Id>$(PackageId)</Id>
				<Version>$(PackageVersion)</Version>
				<DevelopmentDependency Condition="'$(IsDevelopmentDependency)' == 'true'">true</DevelopmentDependency>

				<Authors>$(Authors)</Authors>
				<Owners>$(Owners)</Owners>
				<Title>$(Title)</Title>
				<Description>$(Description)</Description>
				<Summary>$(Summary)</Summary>
				<Language>$(NeutralLanguage)</Language>
				<Copyright>$(Copyright)</Copyright>

				<RequireLicenseAcceptance>$(PackageRequireLicenseAcceptance)</RequireLicenseAcceptance>
				<LicenseUrl>$(PackageLicenseUrl)</LicenseUrl>
				<ProjectUrl>$(PackageProjectUrl)</ProjectUrl>
				<IconUrl>$(PackageIconUrl)</IconUrl>
				<Tags>$(PackageTags)</Tags>

				<ReleaseNotes>$(PackageReleaseNotes)</ReleaseNotes>
				<RepositoryUrl>$(RepositoryUrl)</RepositoryUrl>
				<RepositoryType>$(RepositoryType)</RepositoryType>
				<PackageTypes>$(PackageTypes)</PackageTypes>
</PackageTargetPath>
